03week/ticTacToe.js

```diff
@@ -68,7 +108,7 @@ if (typeof describe === 'function') {
     });
     it('should alternate between players', () => {
       ticTacToe(0, 0);
-      assert.deepEqual(board, [ ['O', ' ', ' '], [' ', 'X', ' '], [' ', ' ', ' '] ]);
+      assert.deepEqual(board, [ ['O', '', ' '], [' ', 'X', ' '], [' ', ' ', ' '] ]);
```

Comment:
Put the space back! The tests need it! (In board[0][1], I think you accidentally took it out).